src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,145 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.exception.DukeException;
+import duke.task.Task;
+
+/**
+ * Manages the user interface and contains all user outputs to print during user interaction.
+ */
+public class Ui {
+    private Scanner s;
+
+    /**
+     * Creates new Ui object.
+     */
+    public Ui() {
+        this.s = new Scanner(System.in);
+        printIntro();
+    }
+
+    /**
+     * Prints logo of DUKE and short introduction.
+     */
+    public void printIntro() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello FROM\n" + logo);
+    }
+
+    /**
+     * Reads commands from text file line by line.
+     *
+     * @return Command from text file.
+     */
+    public String readCommand() {
+        return s.nextLine().trim();
+    }
+
+    /**
+     * Prints cue for user to start user input.
+     */
+    public void start() {
+        System.out.println("Hello! I'm Duke\n");
+        System.out.println("What can I do for you?");
+        System.out.println("__________________________________________");
```

Comment:
Is this meant to have less indentation than the later lines? Could consider abstracting this, in case you decide to change number of spaces later and saves you from copy-pasting!